extensions/amp-jwplayer/amp-jwplayer.md

```diff
@@ -0,0 +1,115 @@
+<!---
+Copyright 2015 Longtail Ad Solutions Inc.
```

Comment:
Are you guys ok to contributing the code to the AMP Project and switching the copyright to " 2016 The AMP HTML Authors. All Rights Reserved." ? The one currently in place won't pass the projects presubmit checks.